tests/common.sh

```diff
@@ -18,7 +18,30 @@ function delete_operator {
   kubectl delete -f resources/operator.yaml
 }
 
+function olm_dep {
```

Comment:
So technically this is the marketplace, not the OLM (the OLM is a dependency of the marketplace, and it is already on the CI systems).

Reply:
ack will update it.